Remove buildtime from binaries to avoid unnecessary change in ELF data.

diff --git unix/vncconfig/buildtime.c unix/vncconfig/buildtime.c
index 3f4c369..2e34d4c 100644
--- unix/vncconfig/buildtime.c
+++ unix/vncconfig/buildtime.c
@@ -15,4 +15,3 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
  * USA.
  */
-char buildtime[] = __DATE__ " " __TIME__;
diff --git a/unix/vncviewer/AboutDialog.h b/unix/vncviewer/AboutDialog.h
index ff7290a..ca6eef1 100644
--- unix/vncviewer/AboutDialog.h
+++ unix/vncviewer/AboutDialog.h
@@ -30,7 +30,6 @@
 #define gettext_noop(String) String
 #define N_(String) gettext_noop (String)
 
-extern char buildtime[];
 
 class AboutDialog : public TXMsgBox {
 public:
diff --git a/unix/vncviewer/InfoDialog.h b/unix/vncviewer/InfoDialog.h
index a95f57b..ffa9531 100644
--- unix/vncviewer/InfoDialog.h
+++ unix/vncviewer/InfoDialog.h
@@ -26,7 +26,6 @@
 #include "TXLabel.h"
 #include "TXButton.h"
 
-extern char buildtime[];
 
 class InfoDialog : public TXDialog, public TXButtonCallback {
 public:
diff --git a/unix/vncviewer/buildtime.c b/unix/vncviewer/buildtime.c
index 3f4c369..2e34d4c 100644
--- unix/vncviewer/buildtime.c
+++ unix/vncviewer/buildtime.c
@@ -15,4 +15,3 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
  * USA.
  */
-char buildtime[] = __DATE__ " " __TIME__;
diff --git a/unix/vncviewer/vncviewer.cxx b/unix/vncviewer/vncviewer.cxx
index e418267..a4326bc 100644
--- unix/vncviewer/vncviewer.cxx
+++ unix/vncviewer/vncviewer.cxx
@@ -131,7 +131,6 @@ IntParameter qualityLevel("QualityLevel",
 
 char aboutText[1024];
 char* programName;
-extern char buildtime[];
 
 static void CleanupSignalHandler(int sig)
 {
@@ -273,7 +272,7 @@ int main(int argc, char** argv)
   bindtextdomain(PACKAGE_NAME, LOCALEDIR);
   textdomain(PACKAGE_NAME);
 
-  const char englishAbout[] = N_("TigerVNC Viewer for X version %s - built %s\n"
+  const char englishAbout[] = N_("TigerVNC Viewer for X version %s\n"
 				 "Copyright (C) 1999-2011 TigerVNC Team and many others (see README.txt)\n"
 				 "See http://www.tigervnc.org for information on TigerVNC.");
 
@@ -281,7 +280,7 @@ int main(int argc, char** argv)
 
   // Write about text to console, still using normal locale codeset
   snprintf(aboutText, sizeof(aboutText),
-	   gettext(englishAbout), PACKAGE_VERSION, buildtime);
+	   gettext(englishAbout), PACKAGE_VERSION);
   fprintf(stderr,"\n%s\n", aboutText);
 
   // Set gettext codeset to what our GUI toolkit uses. Since we are
@@ -292,7 +291,7 @@ int main(int argc, char** argv)
 
   // Re-create the aboutText for the GUI, now using GUI codeset
   snprintf(aboutText, sizeof(aboutText),
-	   gettext(englishAbout), PACKAGE_VERSION, buildtime);
+	   gettext(englishAbout), PACKAGE_VERSION);
 
   rfb::initStdIOLoggers();
   rfb::LogWriter::setLogParams("*:stderr:30");
diff --git a/unix/x0vncserver/buildtime.c b/unix/x0vncserver/buildtime.c
index a96031c..60f511c 100644
--- unix/x0vncserver/buildtime.c
+++ unix/x0vncserver/buildtime.c
@@ -15,4 +15,3 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
  * USA.
  */
-char buildtime[] = __DATE__ " " __TIME__;
diff --git a/unix/x0vncserver/x0vncserver.cxx b/unix/x0vncserver/x0vncserver.cxx
index 64fbe68..be55fa7 100644
--- unix/x0vncserver/x0vncserver.cxx
+++ unix/x0vncserver/x0vncserver.cxx
@@ -53,8 +53,6 @@
 #include <sys/proto.h>
 #endif
 
-extern char buildtime[];
-
 using namespace rfb;
 using namespace network;
 
@@ -369,8 +367,7 @@ char* programName;
 
 static void printVersion(FILE *fp)
 {
-  fprintf(fp, "TigerVNC Server version %s, built %s\n",
-          PACKAGE_VERSION, buildtime);
+  fprintf(fp, "TigerVNC Server version %s\n", PACKAGE_VERSION);
 }
 
 static void usage()
diff --git a/unix/xserver/hw/vnc/buildtime.c b/unix/xserver/hw/vnc/buildtime.c
index 3f4c369..2e34d4c 100644
--- unix/xserver/hw/vnc/buildtime.c
+++ unix/xserver/hw/vnc/buildtime.c
@@ -15,4 +15,3 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
  * USA.
  */
-char buildtime[] = __DATE__ " " __TIME__;
diff --git a/unix/xserver/hw/vnc/xvnc.cc b/unix/xserver/hw/vnc/xvnc.cc
index 5d34f01..f6d6ba7 100644
--- unix/xserver/hw/vnc/xvnc.cc
+++ unix/xserver/hw/vnc/xvnc.cc
@@ -80,7 +80,6 @@ extern "C" {
 #include "randrstr.h"
 #endif /* RANDR */
 #include <X11/keysym.h>
-  extern char buildtime[];
 #if XORG >= 17
 #undef VENDOR_RELEASE
 #undef VENDOR_STRING
@@ -267,7 +266,7 @@ void ddxBeforeReset(void)
 void 
 ddxUseMsg()
 {
-    ErrorF("\nXvnc %s - built %s\n%s", XVNCVERSION, buildtime, XVNCCOPYRIGHT);
+    ErrorF("\nXvnc %s\n%s", XVNCVERSION, XVNCCOPYRIGHT);
     ErrorF("Underlying X server release %d, %s\n\n", VENDOR_RELEASE,
            VENDOR_STRING);
     ErrorF("-screen scrn WxHxD     set screen's width, height, depth\n");
@@ -1290,7 +1289,7 @@ static void vfbClientStateChange(CallbackListPtr*, pointer, pointer) {
 void
 InitOutput(ScreenInfo *screenInfo, int argc, char **argv)
 {
-  ErrorF("\nXvnc %s - built %s\n%s", XVNCVERSION, buildtime, XVNCCOPYRIGHT);
+  ErrorF("\nXvnc %s\n%s", XVNCVERSION, XVNCCOPYRIGHT);
   ErrorF("Underlying X server release %d, %s\n\n", VENDOR_RELEASE,
          VENDOR_STRING);
     int i;
